<?xml version="1.0" ?>
<launch>
  <group ns="restricted_topological_map_generators">
    <node pkg="topological_navigation" name="short" type="restrictions_manager.py" output="screen" respawn="true" >
      <param name="out_topic" type="str" value="short_topological_map"/>
      <remap from="type" to="short_agent_type" />
      <remap from="task" to="short_task_type" />
    </node>
    <node pkg="topological_navigation" type="visualise_map.py" name="visualise_short_map" args="-e" respawn="false">
      <remap from="topological_map" to="short_topological_map"/>
      <remap from="topological_map_visualisation" to="short_topological_vis"/>
    </node>
  </group>
</launch> 
